### ISSUE TITLE
error with csv file implementation

### ISSUE THREAD

**Issue by michelle171** [EESI/Fizzy#5]
I attempted to replicate the csv fizzy implementation example, which included replicating the test documents. I received this error after the same command line call as shown in the example:

fizzy -n 2 -l Class -f MIM -m test.map -i test.csv -s -o test.ouput -r test.biom -v
usage: fizzy [-h] [-n SELECT] [-q BINS] -l LABEL [-f FS_METHOD] -i INPUT_FILE
             [-m MAP_FILE] [-j] -o OUTPUT_FILE [-r OUTPUT_BIOM] [-v]
fizzy: error: unrecognized arguments: -s

When I took out the -s flag, I received this: 
fizzy -n 2 -l Class -f MIM -m test.map -i test.csv -o test.ouput -r test.biom -v
Segmentation fault (core dumped)

I compiled all of the dependencies before trying this example, therefore I am not sure what is going wrong.

**Comment by gditzler** [comment]
Are test.{csv,biom} files that you created? A segmentation fault is likely going to be caused by an issue on the MIToolbox or FEASTtoolbox.

**Comment by michelle171** [comment]
Yes I created those files, replicated from the example for csv files.
I compiled both of those according to their instructions, do you have any suggestions?

**Comment by ghost** [comment]
comiple with debugging flags, run it through gdb

**Comment by yangfenglong** [comment]
> I attempted to replicate the csv fizzy implementation example, which included replicating the test documents. I received this error after the same command line call as shown in the example:
> 
> fizzy -n 2 -l Class -f MIM -m test.map -i test.csv -s -o test.ouput -r test.biom -v
> usage: fizzy [-h] [-n SELECT] [-q BINS] -l LABEL [-f FS_METHOD] -i INPUT_FILE
> [-m MAP_FILE] [-j] -o OUTPUT_FILE [-r OUTPUT_BIOM] [-v]
> fizzy: error: unrecognized arguments: -s
> 
> When I took out the -s flag, I received this:
> fizzy -n 2 -l Class -f MIM -m test.map -i test.csv -o test.ouput -r test.biom -v
> Segmentation fault (core dumped)
> 
> I compiled all of the dependencies before trying this example, therefore I am not sure what is going wrong.

I faced the seam problem with the test data : 
https://github.com/gditzler/DataCollections/blob/master/AmericanGut/AmericanGut-Gut-Sex.biom
https://github.com/gditzler/Fizzy/blob/master/test/Tutorial.md
